src/ImageSequencer.js

```diff
@@ -5,6 +5,7 @@ ImageSequencer = function ImageSequencer(options) {
   options = options || {};
   options.inBrowser = options.inBrowser || typeof window !== 'undefined';
   if (options.inBrowser) options.ui = options.ui || require('./UserInterface');
+  options.initialImage = "";
```

Comment:
Do we need to set it to "" right here? Can't we just start using it, and it will be default undefined (so `options.initialImage !== undefined`) until we fill it? Not a big deal but want to be pretty tidy in this code as it's core functionality. Thanks!